### ISSUE TITLE
Failed to parse header: From: "田中" <sip:100@domain.com>;tag=9m4rgKapFervFSm

### ISSUE THREAD

**Issue by wanwankoko** [DoubangoTelecom/sipml5#291]
Hi i got this problem

Set the "Effective Caller ID Name" or  "effective_caller_id_name" = "Nakata" 
Make Call from that Extension to another
No error  with English name

Set the "Effective Caller ID Name" or  "effective_caller_id_name" = "田中" 
Error with Japanese name

Chrome console output:

Failed to parse header: From: "田中" <sip:100@domain.com>;tag=Ka23r25pFSHNm
tsk_utils_log_error @ SIPml-api.js:1
tsip_header_NameAddr.Parse @ SIPml-api.js:1
tsip_header_parse @ SIPml-api.js:1
tsip_message_parser_execute @ SIPml-api.js:1
tsip_message.Parse @ SIPml-api.js:1
__tsip_transport_ws_onmessage @ SIPml-api.js:1

SIPml-api.js:1 
Uncaught TypeError: Cannot read property 's_tag' of null
    at tsip_dialog_layer.handle_incoming_message (SIPml-api.js:1)
    at tsip_transport_layer.handle_incoming_message (SIPml-api.js:1)
    at WebSocket.__tsip_transport_ws_onmessage (SIPml-api.js:1)

**Comment by n0gard** [comment]
Yes, this issue exists. Because ragel-js cannot handle 2bytes non ascii character right now, but the author of ragel-js did not update it since 8 years ago.
